manifests/config.pp

```diff
@@ -7,7 +7,8 @@
 
   group { $qpid::group:
     ensure => present,
-  } ->
```

Comment:
This dependency should be safe to drop since https://docs.puppet.com/puppet/latest/type.html#user-description states:
> Autorequires: If Puppet is managing the user’s primary group (as provided in the gid attribute) or any group listed in the groups attribute then the user resource will autorequire that group. If Puppet is managing any role accounts corresponding to the user’s roles, the user resource will autorequire those role accounts.